src/layer/S3SLSTRLayer.ts

```diff
@@ -24,7 +24,7 @@ export class S3SLSTRLayer extends AbstractSentinelHubV3Layer {
     view: 'NADIR' | 'OBLIQUE' = 'NADIR',
   ): Promise<PaginatedTiles> {
     const findTilesDatasetParameters: S3SLSTRFindTilesDatasetParameters = {
-      type: this.dataset.shProcessingApiDatasourceAbbreviation,
+      type: this.dataset.datasetParametersType,
```

Comment:
Nice, good that you fixed this - can we maybe just name the `datasetParametersType` so that the name explicitly states where it can be used? Doesn't need to be a short name, maybe sth. like `shSearchIndexV3DatasetType`? (not sure if correct, please check before using)

Reply:
This has moved. But to answer, unfortunately no, can't do such a "personalization" of params which are in all of the dataset objects in `datasets.ts`.

Reply:
No problem, if anyone searches for this they will see where it is used. We can always rename later if it bothers us, resolving.